src/extension/background-script/db.ts

```diff
@@ -25,7 +25,8 @@ class DB extends Dexie {
       blocklist: "++id,host,name,imageURL,isBlocked,createdAt",
     });
     this.version(3).stores({
-      permissions: "++id,allowanceId,host,method,enabled,blocked,createdAt",
+      permissions:
+        "++id,accountId,allowanceId,host,method,enabled,blocked,createdAt",
```

Comment:
you need to add a new version(4) for this.

look here: https://dexie.org/docs/Tutorial/Design#database-versioning

and actually if we don't query the attribute then we would not need to add it - but we want to query it. (also: https://dexie.org/docs/Version/Version.stores()#warning)


check the upgrade function: https://dexie.org/docs/Version/Version.upgrade()
I'd say we simply store the account id of the first account. (`object.keys(accounts)[0]`)

Reply:
I think this has been done in the meantime, I'll resolve this conversation.